analysis/freq.go

```diff
@@ -34,6 +59,14 @@ type TokenFreq struct {
 	frequency int
 }
 
+func (tf *TokenFreq) Size() int {
+	rv := reflectStaticSizeTokenFreq
```

Comment:
Shouldn't we also include len(tf.Term) here?

Reply:
ya good catch